stellarphot/settings/custom_widgets.py

```diff
@@ -985,3 +989,71 @@ def save_wd(_=None):
         raise ValueError(
             f"The widget {setting_widget} is not a recognized type of widget."
         )
+
+
+class PhotometryRunner(ipw.VBox):
+    def __init__(
+        self, photometry_notebook_name="photometry_run.ipynb", *args, **kwargs
+    ):
+        super().__init__(*args, **kwargs)
+        self.photometry_notebook_name = photometry_notebook_name
+        self.fo = FitsOpener(
```

Comment:
`fo` upset the spell checker, so I renamed a few of these to `fitsopen` or something like that in the past.  I can see why you added it to the ignore list.

Reply:
Fixed